src/modules/users/components/GasStation/GasStationHeader/GasStationHeader.tsx

```diff
@@ -40,7 +48,16 @@ const GasStationHeader = ({ close }: Props) => {
           <CopyableAddress>{walletAddress}</CopyableAddress>
         </div>
         <div>
-          <Numeral value={balance} suffix={DEFAULT_NETWORK_TOKEN.symbol} />
+          {isNetworkAllowed && isMobile ? (
```

Comment:
This is not really needed here. I know the spec has it, annoyingly that is just the version of the `GasStationPopover` design that I dragged in. So, just keep it simply the same as the desktop view.

Reply:
@arrenv Have removed and kept same as desktop.